web/src/stream_events.js

```diff
@@ -173,7 +173,7 @@ export function mark_unsubscribed(sub) {
 
         // This update would likely be better implemented by having it
         // disappear whenever no unread messages remain.
-        unread_ui.hide_mark_as_read_turned_off_banner();
+        unread_ui.hide_unread_banner();
```

Comment:
Can you do this renaming as a prep commit? It'll make the actual functional changes way easier to read.

Reply:
I'm also not sure the rest of this commit beyond the renaming makes sense as separated from the final commit, I guess it depends on how you resolve the other feedback.

Reply:
Yeah, after making a separate commit for renaming, it doesn't actually make sense for that additional commit to be separated from the final commit. I'll have it squashed with the final commit.